src/pages/Hojung/Login/Login.js

```diff
@@ -1,9 +1,99 @@
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import './Login.scss';
 
 class LoginHojung extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      btn: false,
+    };
+  }
+
+  componentDidMount() {
+    const token = localStorage.getItem('token');
+    if (token) {
+      this.props.history.push('/main-hojung');
+    }
+  }
+
+  goToMain = e => {
+    e.preventDefault();
+    fetch('http://10.58.3.149:8000/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.id,
+        password: this.state.pw,
+      }),
+    })
+      .then(response => response.json())
+      .then(response => {
+        if (response.access_token) {
+          localStorage.setItem('token', response.access_token);
+          this.props.history.push('/main-hojung', response);
+        } else {
+          alert('아이디 비밀번호를 다시 입력해주세요!');
+        }
+      });
+  };
+
+  handleInput = e => {
+    this.setState({
+      [e.target.name]: e.target.value,
+    });
+  };
+
+  handleKeyPress = e => {
+    const isValid =
+      this.state.id.length >= 1 &&
+      this.state.pw.length >= 5 &&
+      this.state.id.indexOf('@') > -1;
+    this.setState({
+      btn: isValid,
+    });
+  };
+
   render() {
-    return <div>hi</div>;
+    const { handleKeyPress, handleInput, goToMain } = this;
+    const { id, pw, btn } = this.state;
+    return (
+      <div id="login">
+        <div className="loginWrap">
```

Comment:
```suggestion
    const { id, pw, btn } = this.state;
    
    return (
      <div className="login">
        <div className="loginWrap">
```
- 물론 Login 컴포넌트가 하나일 것은 알지만 굳이 Id 셀렉터를 사용할 필요가 일단 없어보입니다. 앱 전체에서 언제나 항상 완벽히 유일하고 고유한... 요소를 상정하는 부담을 지는 것 보다는 그냥 클래스 셀렉터를 쓰는게 맞을 것 같습니다.
- 구분되는 요소들끼리 한줄 씩 띄워주세요

Reply:
수정완료 했습니다